.github/workflows/wasm32.yml

```diff
@@ -35,6 +41,9 @@ jobs:
           tar -f wasm32-wasi-libs.tar.gz -C wasm32-wasi-libs -xz
           rm wasm32-wasi-libs.tar.gz
 
+      - name: Build fresh compiler
+        run: bin/ci with_build_env make # TODO: Remove this after next version update
```

Comment:
This seems to break CI

Reply:
Yeah, we're not using the `bin/ci` wrapper in this job. Just run `make crystal` directly.